go.mod

```diff
@@ -0,0 +1,5 @@
+module github.com/ipfs/go-dnslink
+
+go 1.12
```

Comment:
Let's drop this to go 1.11 for now. We don't _yet_ require 1.12 (I think?).

Reply:
Done, I also changed travis to `1.11` to be consistent.